registered-comuni-pn/src/configs/env.ts

```diff
@@ -0,0 +1,29 @@
+import dotenv from 'dotenv'
+import { z } from 'zod'
+
+export const envSchema = z.object({
+  TENANTS_COLLECTION_NAME: z.string(),
+  PURPOSES_COLLECTION_NAME: z.string(),
+
+  READ_MODEL_DB_USER: z.string(),
+  READ_MODEL_DB_PASSWORD: z.string(),
+  READ_MODEL_DB_HOST: z.string(),
+  READ_MODEL_DB_PORT: z.string(),
+  READ_MODEL_DB_NAME: z.string(),
+
+  SMTP_HOST: z.string(),
```

Comment:
May I suggest changing this to `SMTP_ADDRESS`? 🙏 
Just to keep uniformity through all deployments.

Reply:
Sure!